_posts/zh/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Bitcoin Optech Newsletter #116'
+permalink: /zh/newsletters/2020/09/23/
+name: 2020-09-23-newsletter-zh
+slug: 2020-09-23-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 描述了启用新型手续费提升的软分叉提案，并总结了对永远无法花费的脚本的研究，这些脚本因需要同时满足时间锁和高度锁而无法花费。此外，还包括我们常规的服务和客户端软件更新摘要、新的发布和候选发布列表，以及流行比特币基础设施软件的更改。
+
+## 行动项
+
+*本周无。*
+
+## 新闻
+
+- **<!--transaction-fee-sponsorship-->****交易费用赞助：** Jeremy Rubin 向 Bitcoin-Dev 邮件列表[发布][rubin sponsor bumping]了一项提案，建议通过软分叉添加一种新的手续费提升机制，相较于当前的手续费替代（[RBF][topic rbf]）和子付父（[CPFP][topic cpfp]）方法，该机制可能更不易被滥用。[交易固定][topic transaction pinning]及现有系统中的其他滥用方式可用于攻击合约协议。这些问题促成了如[锚定输出][topic anchor outputs]之类的部分解决方案，但这些方案仅适用于特定协议（如双方的 LN 通道），难以推广，因此不够理想。
+
+  在共识层，Rubin 提议允许交易可选择包含一个特殊的最终输出，该输出承诺一个或多个其他未确认交易的 txid。带有此特殊输出的交易称为*赞助交易*，它只能在其*赞助*的每笔交易也被包含在同一个区块中时才算有效。这意味着希望获得赞助交易高手续费率的矿工将被激励确认低手续费率的赞助交易。除了其特殊的赞助输出，赞助交易是普通的比特币交易。
+
+  在控制内存池接收和交易中继的规则（*策略层*）方面，Rubin 提议了一组简单的更改，允许任何人赞助当前在内存池中的任意交易，或用具有相同承诺且更高手续费率的替代赞助交易替换现有赞助交易。其目的是确保只要赞助交易遵守规则且支付足够高的手续费率，就能在不受固定或其他攻击影响的情况下在网络中传播。
+
+  Rubin 的提案附有一个[参考实现][rubin refimpl]，并讨论了设计权衡和前向兼容性。截止本文发布时，邮件列表中的评论对 Rubin 的工作表示赞赏，但也指出了两点重要复杂性：
+
+  - **<!--floating-payments-still-pinnable-->**浮动支付仍然可能被固定： Antoine Riard [指出][riard heavyweight tx]，即便提案修改为允许赞助特定输入（[更改提案][harding sponsor outpoints]），恶意对手方仍可通过在最大尺寸交易中包含一个使用 `SIGHASH_SINGLE` 签名的特定输入（例如最新 LN 协议中的 HTLC 或 [eltoo][topic eltoo] 状态交易）来固定该输入。
+
+  - **<!--breaks-reorg-safety-guarantee-->**打破了重组安全保障： Suhas Daftuar [提醒][daftuar principle]读者比特币设计的一个基本原则。中本聪曾[写道][nakamoto later block]，“在区块链重组的情况下，交易需要能够进入后续区块中。”然而，赞助交易只能在与其赞助的交易同一区块中才算有效，违反了此原则。
+
+  本提案仍在积极讨论中，我们将在未来期刊中总结任何新的显著讨论。
+
+- **<!--research-into-conflicts-between-timelocks-and-heightlocks-->****对时间锁和高度锁冲突的研究：** Blockstream 工程博客的一篇[文章][blockstream post]描述了比特币时间锁（timelocks）和区块高度锁（heightlocks）之间的相互作用。自比特币初始发布以来，所有交易都有一个 `nLockTime` 字段。一个在区块 31,000（2009 年 12 月）激活的软分叉开始[将][time-height-lock fork]此字段与区块头的显式时间字段及其隐含高度（自创世区块以来的区块数）进行比较。区块仅在其所有交易符合以下两条规则时有效：[^final-sequence]
+
+  - 如果交易的 nLockTime 低于 5 亿，则它必须低于区块高度。
```

Comment:
```suggestion
  - 如果交易的 nLockTime 低于 5 亿，则它必须低于区块高度。

```

The formatting here is off, I think this may solve it. If not, @bittomhan you can try something else (see formatting of en version for comparison)